stories/Containers/DiningReservation.stories.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { mockElement } from '../Tools';
+import { Button } from '../../src';
+
+import { DiningReservation } from '../../src/Containers/DiningReservation';
+
+storiesOf('DiningReservation', module)
+    .addDecorator(withKnobs)
+    .add(
+        'with default',
+        mockElement(
+            modal1 => (
+                <>
+                    <Button
```

Comment:
Use Styled Components?

Reply:
Should the button be in the main component and not in the stories?
I imported the button component

Reply:
The Stories are just to test the Component you built. So the button will appear in both.